### DISCUSSION TITLE
https://blog.3place.cn/3place/lmyn/LMYN044.html

### DISCUSSION THREAD

**Discussion by giscus** [hbaolong/hbaolong.github.io#93] [General]
# https://blog.3place.cn/3place/lmyn/LMYN044.html

https://blog.3place.cn/3place/lmyn/LMYN044.html

**Comment by picklecai** [comment]
国王：跋吉罗陀
恒河：跋吉罗提

**Comment by picklecai** [comment]
这一个恒河下凡的故事；
它带来了吉祥和光辉，
它能使人升天又得子。” （1.43.20）

古印度人两大愿望：升天，得子。

**Comment by picklecai** [comment]
恒河下凡，大概类似于中国人治水，让大江大河为我所用。
即使作者把每个国王都写得同样光辉虔诚能干，这一浩大的工程仍然经历了好几代人。我能想到的类比就是治水。
